flutter/cpp/datasets/README.md

```diff
@@ -126,3 +126,68 @@ you need to replace `squad_eval_mini.tfrecord` by `squad_eval.tfrecord` in the
       --ground_truth_directory=/tmp/ade20k_512/annotations/raw  \
       --num_threads=4
     ```
+
+## SNU SR set
+
+preparing datasets and models
+
+1. dataset:
+
+    1. rotate 1080x1920 ones to 1920x1080 so that all the images are 1920x1080
+
+    2. convert all 1920x1080 jpg images to .png files
```

Comment:
Should it be converting all `*.png` to `*.jpg` files since the code only read `.jpg`, `.jpeg`?